include/deal.II/matrix_free/tools.h

```diff
@@ -157,6 +157,184 @@ namespace MatrixFreeTools
     const unsigned int first_selected_component = 0);
 
 
+
+  /**
+   * A wrapper around MatrixFree to help users to deal with DoFHandlers,
+   * where there are cells without degrees of freedom, i.e., with
+   * FENothing assigned to them. In the following we call such cells
+   * deactivated. All other cells are activated. In contrast to MatrixFree,
+   * this class skips deactivated cells and faces between activated and
+   * deactivated cells are treated as boundary faces.
+   *
+   * The class helps to implement "element birth and death technique",
+   * in which context activated cells are alive and deactivated cells are dead.
```

Comment:
Do you mean
```suggestion
   * in whose context activated cells are alive and deactivated cells are dead.
```
In any case, this does not really say much about what "element birth and death technique" actually is, you just repeat the term in other words (deactivated/activated).